articles/service-fabric/how-to-managed-cluster-modify-node-type.md

```diff
@@ -298,6 +298,66 @@ To modify the OS image used for a node type using an ARM Template, adjust the `v
 }
 ```
 
+## Modify disc size on an existing node type
+Service Fabric Managed Clusters enables you to modify the disc size for new nodes in an exsisting node type in place.  As you deploy and run your application, load increases or decreases may cause a need to change the disc size on a node type.  
+
+[!NOTE]
+While Azure is making updates to the node type, other changes cannot be made to the node type.  Service Fabric Managed Clusters will spin up new nodes with the new disc size; therefore, the old nodes have to be scaled down.  See the below on procedure to scale a node type.
```

Comment:
When Azure is making updates such as X/Y/Z ... 
Second part is confusing, will need to chat with you to understand before I make suggestions